test/FunctionalTests/Microsoft.OData.Client.Tests/Serialization/HttpClientRequestMessageTests.cs

```diff
@@ -18,5 +19,25 @@ public void UnwrapAggregateException()
             var exception = Assert.Throws<DataServiceTransportException>(() => msg.EndGetResponse(task));
             Assert.StartsWith("System.Exception: single exception\r\n", exception.Message);
         }
+
+#if NETCOREAPP2_0_OR_GREATER
+        [Fact]
+        public void SetCertificateValidationCallback()
+        {
+            var httpClientRequestMessage = new HttpClientRequestMessage(
+                new DataServiceClientRequestMessageArgs("GET", new Uri("http://localhost"), false, false, new Dictionary<string, string>()));
+
+            httpClientRequestMessage.ServerCertificateCustomValidationCallback = (_, __, ___, ____) => true;

```

Comment:
any difference between line 30 - 31 and line 36- 37 ?

Reply:
Not really, 36-37 were supposed to be part of the larger 36-39 portion. So, 30-31 is "check always true", 33-34 is "check always false", and 36-39 are "check an aggregation of always true and always false". 36 is really just resetting what we did in 33. I've added some comments to clarify.